pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json

```
@@ -0,0 +1,53 @@
+{
+    "$schema":"https://google.github.io/security-research/kernelctf/metadata.schema.v1.json",
+    "submission_id":[
```

In v2, the property was renamed:
```suggestion
    "submission_ids":[
```